javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,193 @@
+//FLIGHT BOOKING
+function getFullname(firstname, surname, useFormalName = false){
+    if(useFormalName){
+        return 'Lord ' + firstname + ' ' + surname
+    }else{
+        return firstname + ' ' + surname;
+    }   
+}
+
+const fullname1 = getFullname("Juan", "Buendia", true);
+const fullname2 = getFullname("Mario", "Vargas");
+console.log(fullname1);
+console.log(fullname2);
+
+    //to work with gender, add a radio button with the preffix of gender ("Mr, Ms, Mss, Lord")
+    var gender;
+    function getFullNameComplete(genderPreffix = "", firstname, surname, useFormalName = false){
+        gender = genderPreffix
+        if(useFormalName){
+            return gender + ' ' + firstname + ' ' + surname
+        }else{
+            return firstname + ' ' + surname;
+        }   
+    }
+
+    const fullname3 = getFullNameComplete("Mr", "Juan", "Buendia", true);
+    const fullname4 = getFullNameComplete("Lord", "Mario", "Vargas");
+    const fullname5 = getFullNameComplete("Ms", "Maria", "Bonita", true);
+    console.log(fullname1);
+    console.log(fullname2);
+    console.log(fullname3);
+
+//EVENT APPLICATION
+const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+const today = new Date();
+
+function getEventWeekday(day) {
+    var future = (today.getDay() + day) % 7;
+    return days[future];
+}
+
+console.log(getEventWeekday(5)); 
+
+//WEATHER WEAR
+ /* temperature < 0 - winter jacket, mittens and scarf
+ temperature 0 - 10 - jacket and mittens
+ temperature 10 -18 - sweater and jeans
+ temperature 18 -25 - shorts and t-shirt
+ temperature > 25 short, flipflops, light shirt */
+ function showTodaysOutfit(temperature) {
+    switch (true) {
```

Comment:
Would recommend to use _if_ statement here and not _switch_

Reply:
May I ask why? I read when there are several conditional is better to use switch, is there another thing I should consider before choosing between if and switch?

Reply:
Altought your code might work and the JavaScript interpretor allows it, it's not really how you should use a switch...in general you put an _expression_ that can evaluate to value1...value. to be honest I don't even know if your code works since your expression has the the value _true_...of course I might be wrong. Assuming that your code works it's harder to understand versus using an _if_ statement. And most of the time you would write code that your peers need to understand.
`
switch (expression) {
  case value1:
    //Statements executed when the
    //result of expression matches value1
    [break;]
  case value2:
    //Statements executed when the
    //result of expression matches value2
    [break;]
  ...
  case valueN:
    //Statements executed when the
    //result of expression matches valueN
    [break;]
  [default:
    //Statements executed when none of
    //the values match the value of the expression
    [break;]]
}
`

Reply:
I get it. Thanks for the comments. I made the change.